### PR TITLE
LS: move realm specific config to separate files

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

File: BetaSettings.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (supertassu) [inline BetaSettings.php:1]: missing opening php header

File: ProductionSettings.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (supertassu) [inline ProductionSettings.php:1]: this too

**Review by supertassu — CHANGES_REQUESTED** [review]

File: BetaSettings.php (hunk 1 above)
Comment (RhinosF1) [reply BetaSettings.php:2]: done

File: ProductionSettings.php (hunk 2 above)
Comment (RhinosF1) [reply ProductionSettings.php:2]: done

**Comment by JohnFLewis** [comment]
Surely this can just be done on LS as it was before? This seems like a step back in terms of config, especially as LS is managed by ManageWiki, and this new system wouldn’t be.

Wikis can be tagged in this format, it would seem sensible to do that, rather than split config into 3 distinct streams for no reason.

**Comment by Universal-Omega** [comment]
To me this kinda just makes a mess of things. I personally don't see the point in moving out of LS at this time. It seems to be cleaner in LS and 2 additional files don't help with organisation much, and only cause things to be a bit of a mess. Just my opinion though.

**Pull request closed without merging**